robottelo/download_file.py

```diff
@@ -0,0 +1,32 @@
+import os
+import requests
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo import ssh
+
+
+def on_local(url, local_path=None, file_name=None):
+    """ Download file to local from url """
+    _, file_name = file_name or os.path.split(url)
```

Comment:
Extracting file_name single value into two will raise an issue. Fix it.

Reply:
thanks, updated pr.